plugins/modules/pipx.py

```diff
@@ -114,6 +114,12 @@
             - Arbitrary arguments to pass directly to C(pip).
         type: str
         version_added: 4.6.0
+    suffix:
+        description:
+            - Optional suffix for virtual environment and executable names.
```

Comment:
Is this parameter used for idempotency checks?

Reply:
I can read your question in more than one way, could you please be more specific?

Reply:
From the code it is unclear to me whether/how the module determines whether the app is already installed with this suffix, instead of simply be installed with any suffix. The integration tests do not cover this.

Reply:
Hmmm, okie, let me add some tests for that.

Reply:
Can of worms detected :-) Looks like my approach was way too simplistic

Reply:
Sorry for that ;-)

Reply:
What about now? :smile:

Reply:
friendly nudge: @felixfontein

Reply:
Looks ok to me.